src/python/qeqhipster/simulator.py

```diff
@@ -35,7 +35,12 @@ def get_exact_expectation_values(self, circuit, qubit_operator, **kwargs):
         subprocess.call(['/app/zapata/zapata_interpreter_no_mpi_get_exp_vals.out',
                          './temp_qhipster_circuit.txt', str(self.nthreads), './temp_qhipster_operator.txt',
                          './expectation_values.json'])
-        return load_expectation_values('./expectation_values.json')
+        expectation_values = load_expectation_values('./expectation_values.json')
+        term_index = 0
+        for term in qubit_operator.terms:
+            expectation_values.values[term_index] = np.real(qubit_operator.terms[term]*expectation_values.values[term_index])
```

Comment:
Is there any chance someone would want the resulting expectation values to have imaginary parts as well?

Reply:
I don't think so – I think complex expectation values are non-physical. 
Even if one did, it will crash in other places down the road (like serialization), so if we would like to support complex expectation values, it's topic for another story.